function seRompe() {
    return 3 +z;
}


try {
    seRompe()
}catch(err){
    console.error('Se origino un error')
    console.error(err.message)
}